src/database/queries/getMyAns.js

```diff
@@ -0,0 +1,3 @@
+const connection = require('../connection');
+const getMyAns = (consultant_id , answer)=>connection.query(`select * from posts`,[consultant_id, answer]);
+module.exports=getMyAns;
```

Comment:
@denaHS Here's another query which you might want to add a `where` clause to, to only select posts that have been answered by the intended `consultant_id`.